pkg/datastore/applicationstore.go

```diff
@@ -69,17 +71,33 @@ func (s *applicationStore) AddApplication(ctx context.Context, app *model.Applic
 func (s *applicationStore) EnableApplication(ctx context.Context, id string) error {
 	return s.ds.Update(ctx, applicationModelKind, id, applicationFactory, func(e interface{}) error {
 		app := e.(*model.Application)
+		if app.Deleted == true {
```

Comment:
haha, this is what you said.

Reply:
lol

Reply:
oops. This is not what I said. lol. Let me fix this.
What I said is what @khanhtc1202 mentioned at https://github.com/pipe-cd/pipe/pull/1308#issuecomment-749355034